### PR TITLE
[blog] Flink Community Update - August'20

### PR DESCRIPTION
**Pull request by morsapaes**
Slightly delayed, but adding a Community Update blogpost for August 2020.

### CODE REVIEW

**Comment by morsapaes** [comment]
@rmetzger or @sjwiesman, if you can have a look and +1 this, I'd appreciate it.

File: _posts/2020-08-31-community-update.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+* **Fine-grained control over state** for remote functions, including the ability to configure different state expiration modes for each individual function.
+
+As the community around StateFun grows, the release cycle will follow this pattern of smaller and more frequent releases to incorporate user feedback and allow for faster iteration. If you’d like to get involved, we’re always looking for [new contributors](https://github.com/apache/flink-statefun#contributing)!
+
+### Flink 1.10.2
+
+The community has announced the second patch version to cover some outstanding issues in Flink 1.10. You can find a detailed list with all the improvements and bugfixes that went into Flink 1.10.2 in the [announcement blogpost](https://flink.apache.org/news/2020/08/25/release-1.10.2.html).
+
+<hr>
```
Comment (rmetzger) [inline _posts/2020-08-31-community-update.md]: You could have mentioned the Flink 1.11.2 plans as well (or did this discussion start after you wrote the blog?

File: _posts/2020-08-31-community-update.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        <li><b>Support Stateful Python UDFs</b></li>
+        <p>Python UDFs have been supported in PyFlink <a href="https://flink.apache.org/news/2020/02/11/release-1.10.0.html#pyflink-support-for-native-user-defined-functions-udfs">since 1.10</a>, but were so far limited to stateless functions. The community is now looking to introduce stateful aggregate functions (UDAFs) in the Python Table API.</p>
+        <p><b>Note: </b>Pandas UDAFs are covered in a separate proposal (<a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-137%3A+Support+Pandas+UDAF+in+PyFlink">FLIP-137</a>).</p>
+      </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+For a complete overview of the development threads coming up in the project, check the [Flink Wiki](https://cwiki.apache.org/confluence/display/FLINK/Flink+Improvement+Proposals) and follow the feature discussions in the [@dev mailing list](https://lists.apache.org/list.html?dev@flink.apache.org).
```
Comment (rmetzger) [inline _posts/2020-08-31-community-update.md]: You could also link to the list of 1.12 features in the wiki here: https://cwiki.apache.org/confluence/display/FLINK/1.12+Release

File: _posts/2020-08-31-community-update.md (hunk 1 above)
Comment (morsapaes) [reply _posts/2020-08-31-community-update.md]: Yeah, it started after I opened the PR. But for minor releases I usually only include the release itself once it's out.

**Review by rmetzger — APPROVED** [review]
Very nice update! Thank you!

I left some suggestions, but I leave it up to you to decide if you want to push an update or if I should publish it as-is.

File: _posts/2020-08-31-community-update.md (hunk 1 above)
Comment (rmetzger) [reply _posts/2020-08-31-community-update.md]: okay, I see.

**Pull request closed without merging**